test/integration/commands/apply/index.test.ts

```diff
@@ -69,13 +69,17 @@ describe('create command', () => {
     fs.writeFileSync(changesetPath, JSON.stringify(changeset, null, 2))
     fs.writeFileSync(changesetPathAddItems, JSON.stringify(addTwoItemsChangeset, null, 2))
 
-    cdaTokenWithOnlyMasterAccess = await createCdaToken(testSpace, ['master'])
+    try {
+      cdaTokenWithOnlyMasterAccess = await testSpace.getApiKey(INTEGRATION_TEST_KEY_MASTER)
+    } catch (e) {
+      cdaTokenWithOnlyMasterAccess = await createCdaToken(testSpace, ['master'])
```

Comment:
In what case would we end up in the catch clause?

Reply:
if the key is not found, for example, if it was removed